src/main/java/core/basesyntax/Fruit.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Fruit {
+    private static List<Fruit> fruitAvailable = new ArrayList<>();
```

Comment:
this class represents a Fruit
let's store available fruit in storage

Reply:
done